app/controllers/jobs_controller.rb

```diff
@@ -2,7 +2,7 @@ class JobsController < ApplicationController
   before_action :authorize!
 
   def index
-    @jobs = Job.order(posting_date: :desc)
+    @jobs = Job.order(posting_date: :desc).includes(:company)
```

Comment:
added the includes to cut out the n+1 query when we iterate over the jobs and calling job.company